core/src/main/scala/org/apache/spark/storage/BlockManager.scala

```diff
@@ -632,18 +632,7 @@ private[spark] class BlockManager(
    */
   def getLocalBytes(blockId: BlockId): Option[BlockData] = {
     logDebug(s"Getting local block $blockId as bytes")
-    // As an optimization for map output fetches, if the block is for a shuffle, return it
-    // without acquiring a lock; the disk store never deletes (recent) items so this should work
-    if (blockId.isShuffle) {
```

Comment:
Just curious why not delete `isShuffle`? Presume from your PR it can never be true.

Reply:
Thx @attilapiros , now I see.